app/models/organisation.rb

```diff
@@ -25,6 +25,7 @@ class Organisation < ActiveRecord::Base
   validates_presence_of :whitehall_slug
   validates_uniqueness_of :whitehall_slug
   validates_presence_of :title
+  validates_presence_of :content_id
```

Comment:
We should probably also validate uniqueness of `content_id` and have a unique index on it for consistency (I missed that in https://github.com/alphagov/transition/pull/489). [Null values are not considered equal](http://www.postgresql.org/docs/9.3/static/indexes-unique.html) by Postgres so the orgs in the database which no longer exist by slug in Whitehall and therefore won't get their content_ids set on import shouldn't cause problems.